.github/workflows/build_and_test.yml

```diff
@@ -122,9 +132,9 @@ jobs:
       run: cd ./bindings/pylibafl && maturin build
 
   fuzzers:
-    env:
-      CC: ccache clang # use ccache in default
-      CXX: ccache clang # use ccache in default
+  #  env:
```

Comment:
In case we don't want ccache anymore, we should remove the comment, too

Reply:
fix